packages/web3-eth2-beaconchain/package.json

```diff
@@ -10,18 +10,27 @@
     "types": "types.ts",
     "scripts": {
         "test": "jest",
-        "tsc": "babel src --out-dir lib --extensions \".ts,.tsx\" --source-maps inline"
+        "tsc": "babel src --out-dir lib --extensions \".ts,.tsx\" --source-maps inline",
```

Comment:
should this be renamed ?

Reply:
I would like it to be, but some of our CI scripts use `lerna exec tsc`. Perhaps we can open a separate PR to update the scripts/npm commands to `compile` instead of `tsc`?

Reply:
yeah that makes sense!